### PR TITLE
chore: rename image card

### PR DESCRIPTION
**Pull request by luqven**

### CODE REVIEW

**Comment by luqven** [comment]
Current dependencies on/for this PR:
* next
  * **PR #184** 
    * **PR #185** 
      * **PR #193** 
        * **PR #192** 
          * **PR #197** 
            * **PR #194** 
              * **PR #191**  (this PR)
      * **PR #187** 
        * **PR #186** 

This comment was auto-generated by [Graphite](https://graphite.dev?utm_source=gh).

File: frontend/src/components/image/Image.tsx [hunk 1]
```diff
@@ -0,0 +1,36 @@
+import React from "react";
+import Imgix from "react-imgix";
+import { ImgixGETAssetsData } from "../../types";
+
+type Props = {
+  asset: ImgixGETAssetsData[0];
+  domain: string;
+};
+
+export function Image({ asset, domain }: Props) {
```
Comment (frederickfogerty) [inline frontend/src/components/image/Image.tsx:10]: I think this component could have a better name, for me Image is a bit too general. Maybe `AssetCardImage` is a bit better?

**Review by frederickfogerty — APPROVED** [review]
LGTM apart from the name

File: frontend/src/components/image/Image.tsx (hunk 1 above)
Comment (luqven) [reply frontend/src/components/image/Image.tsx:10]: Fixing this is a follow-up PR. Gt stack became corrupted locally 😢

**Pull request merged**